sklearn/naive_bayes.py

```diff
@@ -115,9 +115,15 @@ class GaussianNB(BaseNB):
 
     Read more in the :ref:`User Guide <gaussian_naive_bayes>`.
 
+    Parameters
+    ----------
+    priors : array-like, size=(n_classes,)
+        Prior probabilities of the classes. If specified the priors are not
+        adjusted according to the data.
+
     Attributes
     ----------
-    class_prior_ : array, shape (n_classes,)
+    priors_ : array, shape (n_classes,)
```

Comment:
you cannot do this. It breaks the API and it would require a full deprecation cycle... I don't think the feature justifies such a deprecation.